examples/with-prisma/packages/database/src/client.ts

```diff
@@ -1,6 +1,7 @@
 import { PrismaClient } from "@prisma/client";
 
 declare global {
+  // eslint-disable-next-line no-unused-vars
   var prisma: PrismaClient | undefined;
```

Comment:
hmm, is this necesssary?

Reply:
Define necessary.

Reply:
heh, sorry, I meant is that eslint disabling necessary. What is this declare block is doing?

Reply:
I guess you removed it already :)